Source/SwiftLintFramework/Rules/RuleConfigurations/NameConfiguration.swift

```diff
@@ -17,6 +17,8 @@ public struct NameConfiguration: RuleConfiguration, Equatable {
     var minLength: SeverityLevelsConfiguration
     var maxLength: SeverityLevelsConfiguration
     var excluded: Set<String>
+    var allowedSymbols: Set<String>
```

Comment:
I was thinking more about keeping them consistente since they all use the same configuration type